src/doc/unstable-book/src/language-features/const-fn.md

```diff
@@ -6,24 +6,5 @@ The tracking issue for this feature is: [#57563]
 
 ------------------------
 
-The `const_fn` feature allows marking free functions and inherent methods as
-`const`, enabling them to be called in constants contexts, with constant
-arguments.
-
-## Examples
-
-```rust
-#![feature(const_fn)]
-
-const fn double(x: i32) -> i32 {
-    x * 2
-}
-
-const FIVE: i32 = 5;
-const TEN: i32 = double(FIVE);
-
-fn main() {
-    assert_eq!(5, FIVE);
-    assert_eq!(10, TEN);
-}
-```
+The `const_fn` allows to enable additional functionality
+to already stabilized [minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)
```

Comment:
```suggestion
`const_fn` feature enables additional functionality not stabilized in the
[minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)
```
Would this be clearer?

Reply:
I think we should keep the `The` in front, but otherwise, yes.

Reply:
I have that originally but I removed it.
```suggestion
The `const_fn` feature enables additional functionality not stabilized in the
[minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)
```